synapse/models/orgs.py

```diff
@@ -92,18 +92,42 @@ def getBaseModels():
                 ('ou:sic', {'subof': 'int', 'doc': 'Standard Industrial Classification Code'}),
                 ('ou:naics', {'subof': 'int', 'doc': 'North American Industry Classification System'}),
 
-                ('ou:suborg',
-                 {'subof': 'comp', 'fields': 'org,ou:org|sub,ou:org', 'doc': 'An org which owns a sub org'}),
-                ('ou:member', {'subof': 'comp', 'fields': 'org,ou:org|person,ps:person',
-                               'doc': 'A person who is (or was) a member of an organization.'}),
+                ('ou:suborg', {
+                    'subof': 'comp',
+                    'fields': 'org,ou:org|sub,ou:org',
+                    'doc': 'An org which owns a sub org'}),
+
+                ('ou:member', {
+                    'subof': 'comp',
+                    'fields': 'org,ou:org|person,ps:person',
+                    'doc': 'A person who is (or was) a member of an organization.'}),
 
                 ('ou:hasalias', {'subof': 'comp', 'fields': 'org=ou:org,alias=ou:alias'}),
+
                 ('ou:org:has', {
                     'subof': 'xref',
                     'source': 'org,ou:org',
                     'doc': 'An org owns, controls, or has exclusive use of an object or resource,'
                         'potentially during a specific period of time.'}),
 
+                ('ou:meet', {
```

Comment:
Should this also have name and time fields like ou:conference?

Reply:
ou:conference has a field `date`, not time, I was wrong. ou:conference doesn't have a subprop for date. Should we add it?